### PR TITLE
[FEAT] add Multiple Physics Pretraining (MPP) model

### PR DESCRIPTION
**Pull request by lkyu-ly**
- integrate AViT (Axial Vision Transformer) with MPP normalization into `ppcfd.models`                                                                           
- port all model architecture, custom optimizers (Adan, DAdapt), and data loaders from PaddlePaddle migration                                                  
- add training, inference, and weight conversion scripts under `examples/`                                                                                       
- support model variants Ti/S/B/L with PDEBench multi-physics datasets (SWE, NS, diffusion-reaction)  

This example is able to run on both Nvidia GPU and Custom Device(using iluvatar for test). CINN is also supported.

forward precision:
```text
max_abs_error: 7.8231096268e-07
mean_abs_error: 9.9473159783e-08
rmse: 1.3069938825e-07
max_rel_error: 4.3364424808e-03
mean_rel_error: 6.9402936241e-06
```

### CODE REVIEW

File: examples/multiple_physics_pretraining/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+Multiple Physics Pretraining is a pretraining strategy in which multiple sets of dynamics are jointly normalized and embedded into a single space for prediction. It uses an **AViT** (Axial Vision Transformer) architecture that learns multiple physics simultaneously through pretraining, enabling strong finetuning performance even across different physics domains.
+
+Paper: "Multiple Physics Pretraining for Spatiotemporal Surrogate Models" (NeurIPS 2024)
+
+Below are quick instructions on paddle, full readme please visit https://github.com/PolymathicAI/multiple_physics_pretraining
+
+## Installation
+
+```bash
+pip install ppcfd
```
Comment (zhiminzhang0830) [inline examples/multiple_physics_pretraining/README.md]: 这个应该是 pip install -e . 命令更合理吧，辛苦再验证下安装步骤

File: examples/multiple_physics_pretraining/README.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+### Import the model
+
+```python
+from ppcfd.models.multiple_physics_pretraining import AViT, build_avit
+```
+
+### Train (single device)
+
+```bash
+python train_basic.py --run_name my_experiment --config basic_config --yaml_config config/mpp_avit_ti_config.yaml
```
Comment (zhiminzhang0830) [inline examples/multiple_physics_pretraining/README.md:29]: Image

数据集的下载也需要在文档中写明，辛苦补充下

**Review by zhiminzhang0830 — COMMENTED** [review]
辛苦再完善一下安装、运行的文档，谢谢

File: examples/multiple_physics_pretraining/README.md (hunk 1 above)
Comment (lkyu-ly) [reply examples/multiple_physics_pretraining/README.md]: 是的，此处为遗漏，后续在aistudio的项目修正了，感谢指出
https://aistudio.baidu.com/projectdetail/10079567

File: examples/multiple_physics_pretraining/README.md (hunk 2 above)
Comment (lkyu-ly) [reply examples/multiple_physics_pretraining/README.md:29]: 好的，PDEBench的数据集由于太过庞大无法直接验证，实际使用的是经由PDEBench项目代码生成的较小型swe数据集`PDEBench/data/2D_rdb_NA_NA/2D_rdb_NA_NA.h5`，在本模型中的定义方法是修改位于`examples/multiple_physics_pretraining/config/mpp_avit_b_config.yaml`的`train_data_paths`和`valid_data_paths`.

Image

Comment (zhiminzhang0830) [reply examples/multiple_physics_pretraining/README.md:29]: 那补充下这个小型数据集的下载方式？

Comment (lkyu-ly) [reply examples/multiple_physics_pretraining/README.md:29]: 已测试原版数据集可行性，并在readme中写明
